mteb/tasks/Classification/multilingual/IndicSentimentClassification.py

```diff
@@ -83,10 +83,10 @@ def dataset_transform(self) -> None:
         # Convert to standard format
         for lang in self.hf_subsets:
             self.dataset[lang] = self.dataset[lang].filter(
-                lambda x: x["LABEL"] is not None
+                lambda x: x["label"] is not None
```

Comment:
Why did you update this? it should remain "LABEL" because it's the column name in the original dataset
```suggestion
                lambda x: x["LABEL"] is not None
```